src/main/java/mServer/crawler/sender/br/BrCrawler.java

```diff
@@ -0,0 +1,115 @@
+package mServer.crawler.sender.br;
+
+import de.mediathekview.mlib.Config;
+import de.mediathekview.mlib.Const;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'de.mediathekview.mlib.Const'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)